features/networking/ovn.feature

```diff
@@ -0,0 +1,86 @@
+Feature: OVN related networking scenarios
+
+  # @author rbrattai@redhat.com
+  # @case_id OCP-26139
+  @admin
+  @destructive
+  Scenario: Repeat delete and creation
+  #Test for bug https://bugzilla.redhat.com/show_bug.cgi?id=1781297
+    Given I have a project
+    And evaluation of `project.name` is stored in the :usr_project clipboard
+
+    Given I run the steps 4 times:
+    """
+    Given I have a pod-for-ping in the "<%= cb.usr_project %>" project
+    Given I store the ovnkube-master "south" leader pod in the clipboard
+    When admin deletes the ovnkube-master "south" leader
```

Comment:
@pruan-rht originally I tried to use the clipboard here.  

```
    When I store the ovnkube-master "south" leader pod in the clipboard
    Then the step should succeed
    Given admin ensures "<%= cb.south_leader.name %>" pod is deleted from the "openshift-ovn-kubernetes" project

```

but the `<%= cb.south_leader.node_name %>` failed inside the loop because the clipboard wasn't set yet.

It looks like `transform.rb` does the interpolation
https://github.com/openshift/verification-tests/blob/master/features/step_definitions/transform.rb#L15
but it isn't setup to delay interpolation inside loops?

Is there a way to use `<% %>` inside inner loops?   The workaround seems to be combining steps and using the clipboard inside the steps.

Reply:
you use `#{expression}` inside loop steps

Reply:
ah ha.  that will make life easier.

Reply:
`#{}` doesn't seem to work.  

```
    And I wait up to 30 seconds for the steps to pass:
    """
    When I store the ovnkube-master "south" leader pod in the :post_split_south_leader clipboard using node "#{cb.south_leader.node_name}"
```
The `#{cb.south_leader.node_name}` doesn't get interpolated. 

I see in meta_steps.rb we call `steps` with the step as a string.

```
    success = wait_for(seconds) {
      # repetitions += 1
      # this message needs to be disabled as it defeats deduping
      # logger.info("Beginning step repetition: #{repetitions}")
      begin
        steps steps_string
        true
```

Maybe we could re-interpolate there, before calling `steps steps_string`

The main use case was to set a variable into a clipboard inside the loop step, then interpolate that new clipboard value inside the same loop step.

Reply:
This fell under the cracks, sorry.  Could you full log of how it fails. I couldn't reproduce any issue with expression evaluation. See #1289